test/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "env": {
+    "mocha": true
+  }
+}
```

Comment:
I use [ESLint configuration cascading](]http://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy)

Do you prefer enable mocha env in whole project ? Or define env mocha with inline comment `/*eslint-env  mocha */` in test/index.js